src/fluree/db/peer/http_api.clj

```diff
@@ -814,10 +819,30 @@
   (fn [request]
     (let [header-map (reduce (fn [m [k v]] (assoc m (name k) v))
                              {} (partition 2 headers))
-          response (handler request)]
+          response   (handler request)]
       (update response :headers merge header-map))))
 
 
+(defn command-handler [system {:keys [headers body] :as _request}]
+  (let [start        (System/nanoTime)
+        cmd          (decode-body body :json)
+        ledger       (some-> cmd :ledger keyword)
+        timeout      (or (:request-timeout headers) 60000)
+        opts         {:timeout timeout}
+        _            (log/info "Sending command:" (pr-str cmd))
+        [headers body] (<?? (action-handler :command system cmd nil ledger
+                                            opts))
+        request-time (- (System/nanoTime) start)]
+    (log/info "Command result:" (pr-str body))
```

Comment:
Do we want to log this at the info level for every request?

Reply:
Nah, probably not. I'll change it to debug.